modules/net-vpc-peering/variables.tf

```diff
@@ -59,3 +59,9 @@ variable "routes_config" {
   nullable = false
   default  = {}
 }
+
+variable "stack_type" {
```

Comment:
Can you add a validation rule to make sure only allowed values can be specified, and default to `null`?

Reply:
Hi @ludoo I will add the validation for the possible values. But for the null part, it says in the documentation that the default value for stack_type is "IPV4_ONLY". Should I still keep it the default as null?

Reply:
In general, when the provider has a default value we pass `null` so we defer to the provider by default.